operator/v1/types_network.go

```diff
@@ -116,9 +169,21 @@ type AdditionalNetworkDefinition struct {
 	// If not given the network will be created in the default namespace.
 	Namespace string `json:"namespace,omitempty"`
 
+	// Ipam is the type of IPAM module will be used for IP Address Management(IPAM).
```

Comment:
I'd rather see the IPAM section in the SimpleMacVLAN object. It can be reused by other networks that have a similar ipam story.

Reply:
I've discuss about it with @dcbw and he said that it could be better outside of SimpleMacVlan object because IPAM structure will be shared among other interface type.

@dcbw what do you think about it? move it to SimpleMacVlan?

Reply:
Then they can also embed the ipam structs. But adding at the top-level implies that all networks support it.